packages/react-reconciler/src/__tests__/ReactSuspenseFuzz-test.internal.js

```diff
@@ -32,6 +32,8 @@ describe('ReactSuspenseFuzz', () => {
     Random = require('random-seed');
   });
 
+  jest.setTimeout(20000);
```

Comment:
What is this for?

Reply:
On CI it sometimes times out because it runs so many test cases